addons/mail/static/src/discuss/core/public_web/thread_actions.js

```diff
@@ -8,7 +8,7 @@ threadActionsRegistry.add("show-threads", {
     close(component, action) {
         action.popover?.close();
     },
-    condition: (component) => component.thread?.hasSubChannelFeature,
+    condition: (component) => component.thread?.channel_type === "channel",
```

Comment:
We could still rely on `hasSubChannelFeature`. Maybe something like:
```suggestion
    condition: (component) => component.thread?.hasSubChannelFeature || component.thread?.parent_channel_id?.hasSubChannelFeature
```